modules/docs/arrow-docs/docs/docs/typeclasses/traverse/README.md

```diff
@@ -9,8 +9,259 @@ permalink: /docs/typeclasses/traverse/
+
+
+## Sequence
+
+When we want to traverse a collection that each of its elements already contains an effect, for example, List<Option<A>>, we may want to convert it to Option<List<A>> to work easily with the elements. To do that we could traverse the list applying the identity function.
+
+```kotlin:ank
+val listofOptionalNumbers: List<Option<Int>> = listOf(Option(1), Option(2), Option(3))
+
+listofOptionalNumbers.traverse(Option.applicative(), ::identity)
+```
+
+The equivalent to do a traverse applying identity is using sequence.
+
+```kotlin:ank
+val sequenceOptions = listofOptionalNumbers.sequence(Option.applicative())
+```
+
+We could also use sequence on a list of Either.
+
+```kotlin:ank
+val listOfEither: List<Either<NumberFormatException, Int>> = listOfValidNumbers.map { s -> parseIntEither(s) }
+
+listOfEither.sequence(Either.applicative<NumberFormatException>())
+```
+
+## traverse_ and sequence_
+
+Another usage for sequence is when we are traversing a list of data to which we apply some effectful function and we do not care about the returned values.
+
+Continuing with our first example, imagine a function saveProfile that perform a side effect, saving a profile in a database, and returning Unit asynchronously.
+
+```kotlin:ank
+fun saveProfile(user: User): DeferredK<Unit> = ValidEffects.savingProfiles()
+```
+
+If we apply traverse, we will have an Asynchronous computation result with a List of Unit that we do not care about.
+
+```kotlin:ank
+fun saveProfiles(users: List<User>): Kind<ForDeferredK, Kind<ForListK, Unit>> = users.traverse(DeferredK.applicative(), { user -> saveProfile(user)})
+```
+
+We would just prefer to have an Unit as a result since that would convey the same information.
+
+Traversing solely for the sake of the effect (ignoring any values that may be produced, Unit or otherwise) is common, so Foldable (superclass of Traverse) provides `traverse_` and `sequence_` methods that do the same thing as `traverse` and `sequence` but ignores any value produced along the way, returning Unit at the end.
+
+```kotlin:ank
+val listOfUsers = listOf(User("1","Jimmy"), User("2","Peter"), User("3","Rob"))
+val result: Kind<ForDeferredK, Kind<ForListK, Unit>> = saveProfiles(listOfUsers)
+result.fix().unsafeRunSync()
+```
+
+In the example above, result will hold an asynchronous computation with a list of effectful results, a list of Unit.
+
+result.fix().unsafeRunSync()) is just to force the computation of the asynchronous operation and see the result of the example.
+
+That should return a ListK(list=[kotlin.Unit, kotlin.Unit, kotlin.Unit])
+
+Let's see what we would get with `traverse_`:
+
+```kotlin:ank
+val l = listOfUsers.traverse_(DeferredK.applicative(), {user -> saveProfile(user)})
+l.fix().unsafeRunSync()
+```
+
+That should return kotlin.Unit
+
+Now, if we have a list already containing effects with results we do not care about: listOfAsyncResults, we can apply `sequence_` to aggregate the result.
```

Comment:
Missing data type table and hierarchy diagram. See docs in other type classes as an example.

Reply:
I have seen that they have been added automatically after running the ank task